dspace-api/src/main/java/org/dspace/app/statistics/GetGithubRelease.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.statistics;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+import javax.inject.Inject;
+
+import org.dspace.scripts.DSpaceRunnable;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+/**
+ * Tool to fetch a release from a Github repository.
+ *
+ * @author mwood
+ */
+public class GetGithubRelease
+        extends DSpaceRunnable<GetGithubReleaseScriptConfiguration> {
+    @Inject
+    GetGithubReleaseScriptConfiguration configuration;
+
+    private boolean verbose;
+
+    Path archiveFilePath;
+
+    @Override
+    public GetGithubReleaseScriptConfiguration getScriptConfiguration() {
+        return configuration;
+    }
+
+    @Override
+    public void setup() {
+        // XXX This method intentionally left blank.
+    }
+
+    @Override
+    public void internalRun()
+            throws Exception {
+        // If asked, display help and exit.
+        if (commandLine.hasOption(GetGithubReleaseOptions.OPT_HELP)) {
+            printHelp();
+            return;
+        }
+
+        // Not help.  Remember whether verbose was required.
+        verbose = commandLine.hasOption(GetGithubReleaseOptions.OPT_VERBOSE);
+
+        // Identify the latest release.
+        String owner = commandLine.getOptionValue(GetGithubReleaseOptions.OPT_OWNER);
+        String repo = commandLine.getOptionValue(GetGithubReleaseOptions.OPT_REPO);
+        URL releaseConnection = new URL(String.format("https://api.github.com/repos/%s/%s/releases/latest",
```

Comment:
Do we want to blindly always gather the latest version?  Or should we really just be taking in a URL here as a parameter... that way it's possible to use a specific tagged version of the files (which could be useful if one version of DSpace supports a different file format than another)

If you feel this is the best approach, I'm OK with it.  I just wanted to point out that blindly taking the "latest" can mean that you might try to download an update for DSpace 7 only to find that the new "latest" version is only compatible with DSpace 8 (as an example)... I don't know if this would ever happen, but wanted to point it out.